multichain-testing/tools/e2e-tools.js

```diff
@@ -106,20 +105,34 @@ export const makeBlockTool = ({ rpc, delay }) => {
  * @param {string} [opts.bundleId]
  */
 const installBundle = async (fullPath, opts) => {
-  const { id, agd, progress = console.log } = opts;
+  const { id, agd, delay, progress = console.log } = opts;
   const { chainId = 'agoriclocal', installer = 'faucet' } = opts;
-  const from = await agd.lookup(installer);
+  const from = agd.lookup(installer);
   // const explainDelay = (ms, info) => {
   //   progress('follow', { ...info, delay: ms / 1000 }, '...');
   //   return delay(ms);
   // };
   // const updates = follow('bundles', { delay: explainDelay });
   // await updates.next();
-  const tx = await agd.tx(['swingset', 'install-bundle', `@${fullPath}`], {
-    from,
-    chainId,
-    yes: true,
-  });
+  const installBundle = () =>
+    agd.tx(['swingset', 'install-bundle', `@${fullPath}`], {
+      from,
+      chainId,
+      yes: true,
+    });
+  let tx;
+  try {
+    tx = await installBundle();
+  } catch (err) {
```

Comment:
Nice, did you consider implementing this logic in `agd.tx` instead? The immediate concern is that this change works, but then `runCoreEval` starts to flake.

We could specifically look for "incorrect account sequence", taking a page from the inter oracle and ocw implementation: https://github.com/SimplyStaking/agoric-ocw/blob/9b1937cea0e573c9488389d48738614419f5a148/src/submitter.ts#L108

Reply:
Good point. I didn't want to put retry logic that low but it didn't occur to me to simply bump the sequence number. I'll work on that now but let this merge.